kousa/lib/onion/room_session.ex

```diff
@@ -37,8 +37,8 @@ defmodule Onion.RoomSession do
            Onion.RoomSessionDynamicSupervisor,
            {__MODULE__, Keyword.merge(initial_values, callers: callers)}
          ) do
-      {:error, {:already_started, pid}} -> {:ok, pid}
-      x -> x
+      {:error, {:already_started, pid}} -> {:ignored, pid}
```

Comment:
@benawad decided to go with {:ignored, pid}, we might still want the pid somehow, this is more in line with the way that you use :ignore with GenServers (especially in the distributed genserver context).